ui/components/src/TimeChart/TimeChart.tsx

```diff
@@ -284,29 +297,34 @@ export const TimeChart = forwardRef<ChartInstance, TimeChartProps>(function Time
           return;
         }
 
-        // Clear previously set pinned crosshair
-        const isCrosshairPinned = seriesMapping[seriesMapping.length - 1]?.name === PINNED_CROSSHAIR_SERIES_NAME;
-        if (tooltipPinnedCoords !== null && isCrosshairPinned) {
-          seriesMapping.pop();
-        } else if (seriesMapping.length !== data.length + 1) {
-          // Only add pinned crosshair line series when there is not one already in seriesMapping.
-          const pinnedCrosshair = merge(DEFAULT_PINNED_CROSSHAIR, {
-            markLine: {
-              data: [
-                {
-                  xAxis: pointInGrid[0],
-                },
-              ],
-            },
-          });
-          seriesMapping.push(pinnedCrosshair);
-        }
+        // Allows user to opt-in to multi tooltip pinning when Ctrl or Cmd key held down
+        const isControlKeyPressed = e.ctrlKey || e.metaKey;
```

Comment:
I think we also use the shift key for similar-ish behavior in some other parts of the codebase (e.g. legends). It might be worth including it here for consistency.

Reply:
I think we should choose one or the other, maybe even go back and change legend selecting to be only the one we decide on (before too many people get used to the current behavior).

Reply:
In the future, 'Shift + Click' might be used for another action like giving users more control over unpinning after multiple tooltips are pinned or possibly adding a user-defined annotation

Reply:
I don't have a strong opinion but they could serve slightly different purposes. Spreedsheets and File browsers use `Cmd + Click` to select multiple items. When there is a list `Shift + Click` selects all items between the first selected and last item. I would try to stay consistent with other applications as much as possible. ([source](https://forums.cronometer.com/discussion/3369/website-functionality-shift-click-vs-ctrl-click))

<img width="212" alt="image" src="https://github.com/perses/perses/assets/6935733/8fb92f6e-a0eb-4906-b893-538818022133">
<img width="308" alt="image" src="https://github.com/perses/perses/assets/6935733/2fe3e40a-d8bc-49ed-8d9a-bd7020ec98fe">

Reply:
I think the legend behavior was set that way to be consistent with previous legend behavior in our dashboards. Maybe this a good question for a PM to help sort out what we want across the board? I'm cool with whatever y'all think is best to help get this PR in for now, and we can always do some adjustments later.